rainier-core/src/main/scala/com/stripe/rainier/core/Discrete.scala

```diff
@@ -1,9 +1,15 @@
 package com.stripe.rainier.core
 
-import com.stripe.rainier.compute.Real
+import com.stripe.rainier.compute.{If, Real}
 
-trait Discrete extends Distribution[Int] {
+trait Discrete extends Distribution[Int] { self: Discrete =>
```

Comment:
Is there any objections to this `self: Discrete =>` reference?

Reply:
Nope, totally fine. (Would also be fine without the type annotation).